verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -282,11 +301,40 @@ function RowMenu(props: {
                         <span className={classes.menuItemTitle}>Edit</span>
                     </MenuItem>
                 </Link>
-                <MenuItem onClick={handleDelete}>
+                <MenuItem onClick={openDeleteDialog}>
                     <DeleteIcon />
                     <span className={classes.menuItemTitle}>Delete</span>
                 </MenuItem>
             </Menu>
+            <Dialog
+                open={deleteDialogOpen}
+                onClose={(): void => setDeleteDialogOpen(false)}
+                onClick={(e): void => e.stopPropagation()}
+            >
+                <DialogTitle>
+                    Are you sure you want to delete this case?
+                </DialogTitle>
+                <DialogContent>
+                    <DialogContentText>
+                        Case {props.rowId} will be permanently deleted and can
+                        not be recovered.
+                    </DialogContentText>
+                </DialogContent>
+                <DialogActions>
+                    <Button
+                        onClick={(): void => {
+                            setDeleteDialogOpen(false);
+                        }}
+                        color="primary"
+                        autoFocus
+                    >
+                        Cancel
+                    </Button>
+                    <Button onClick={handleDelete} color="primary">
```

Comment:
ditto, also shouldn't be the same color as the other button no? perhaps primary and secondary are better.

Reply:
Usually the YES button would be filled to draw attention to it. But when it’s a confirmation dialog you don’t want to encourage them to click YES. https://material.io/components/dialogs#confirmation-dialog